rel8-jdk8-overrides.yaml

```diff
@@ -0,0 +1,41 @@
+schema_version: 1
+
+name: "jboss-eap-7/eap74-openjdk8-openshift-rhel8"
+
+labels:
+    - name: "com.redhat.component"
+      value: "jboss-eap-74-openjdk8-builder-openshift-rhel8-container"
+
+modules:
+      install:
+          - name: jboss.container.openjdk.jdk
+            version: "8"      
+          - name: jboss.container.eap.galleon.build-settings
+            version: "osbs"
+          - name: jboss.container.maven.module
+            version: "3.6"
+          - name: os-eap-python
+            version: '3.6'
+          - name: jboss.container.jolokia
+            version: '7'
+          - name: jboss.container.prometheus
+            version: '7'
+
+packages:
+  manager: dnf
+  content_sets_file: content_sets_rhel8.yml
+
+osbs:
+  configuration:
+    container:
+      platforms:
+        only:
+          - x86_64
+          #- s390x
+          #- ppc64le
+      compose:
+        pulp_repos: true
+        signing_intent: release
+  repository:
+    name: containers/jboss-eap-74-openjdk8-builder
```

Comment:
This looks liek it will overwrite https://github.com/jboss-container-images/jboss-eap-7-openshift-image/blob/eap74-dev/rel-jdk8-overrides.yaml#L42, is this repo name the same as the one created in comet?

Reply:
Yes is the same repo, now we have jb-eap-7.4-rhel- and jb-eap-7.4-rhel-8 branches

Reply:
Ah, right yeah, forgot about that part, +1 from me.